publish/publish.proj

```diff
@@ -185,8 +193,7 @@
   </Target>
   
   <Target Name="PublishDebToolPackageToFeed"
-          Condition="'$(PublishDebToolToFeed)' == 'true'">
-    <Error Condition="'$(CliNuGetFeedUrl)' ==''" Text="Missing required property CliNuGetFeedUrl" />
+          Condition="'$(PublishDebToolToFeed)' == 'true' AND '$(CliNuGetFeedUrl)' != ''">
```

Comment:
Should we promote `PublishDebToolToFeed` and make a `PB_PublishDebToolToFeed` that gets passed in since this already has a mechanism to skip?  Not opposed to doing it this way either.